src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.scss

```diff
@@ -0,0 +1,129 @@
+@import "../../../../../assets/stylesheets/custom";
+@import "../../../../../assets/stylesheets/mixins";
+@import "../../../../../assets/stylesheets/color-variables";
```

Comment:
@michaelkleinhenz @joshuawilson I believe these can all be contained within the parent scss file and still compile properly? Seems like a large amount of subnavs to go through...